app/renderer/actions/Session.js

```diff
@@ -552,14 +552,19 @@ export function setSavedServerParams () {
 
 export function getRunningSessions () {
   return (dispatch, getState) => {
+    const avoidServerTypes = ['sauce', 'testobject'];
```

Comment:
Why is it only Sauce being excluded in this?

Reply:
I didn't change this logic. (Just switched to use `includes` method.)
Ordinary, you implemented this logic. https://github.com/appium/appium-desktop/pull/196
Can I remove here?

Reply:
Ok cool. That's fine then.